package.json

```diff
@@ -208,18 +211,18 @@
     "@typescript-eslint/parser": "5.62.0",
     "@vue/compiler-sfc": "3.3.4",
     "@webb-tools/anchors": "0.5.47",
-    "@webb-tools/api": "0.1.4-126",
-    "@webb-tools/api-derive": "0.1.4-126",
-    "@webb-tools/app-util": "0.1.4-126",
+    "@webb-tools/api": "0.1.4-127",
+    "@webb-tools/api-derive": "0.1.4-127",
+    "@webb-tools/app-util": "0.1.4-127",
     "@webb-tools/contracts": "0.5.47",
     "@webb-tools/interfaces": "0.5.47",
-    "@webb-tools/sdk-core": "0.1.4-126",
+    "@webb-tools/sdk-core": "0.1.4-127",
     "@webb-tools/tangle-substrate-types": "^0.4.4",
-    "@webb-tools/test-utils": "0.1.4-126",
+    "@webb-tools/test-utils": "0.1.4-127",
     "@webb-tools/tokens": "1.0.8",
     "@webb-tools/utils": "0.5.39",
     "@webb-tools/vanchor-client": "0.1.26",
-    "@webb-tools/wasm-utils": "0.1.4-126",
+    "@webb-tools/wasm-utils": "0.1.4-127",
```

Comment:
Are we still using this package at all? Maybe we can remove.

Reply:
I will create another issue for this; we can address it later.